manage_arkime.py

```diff
@@ -45,6 +46,15 @@ def destroy_demo_traffic(ctx):
     cmd_destroy_demo_traffic(profile, region)
 cli.add_command(destroy_demo_traffic)
 
+@click.command(help="Creates an Arkime Cluster in your account")
+@click.option("--name", help="The name you want your Arkime Cluster and its associated resources to have")
```

Comment:
I wonder if --name should be required, otherwise maybe document that "None" will be used

Reply:
Ah, yeah, it should be required.  Will add that in the next PR.